### PR TITLE
[18.0][MIG] product_net_weight

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

**Review by luc-adhoc — APPROVED** [review]
functional review

File: product_net_weight/models/product_product.py [hunk 1]
```diff
@@ -4,23 +4,35 @@
...
+            if (
+                not float_is_zero(product.weight, precision_digits=prec)
+                and float_compare(
+                    product.net_weight, product.weight, precision_digits=prec
+                )
+                > 0
+            ):
                 raise ValidationError(
-                    _("The net weight of product must be lower than gross weight.")
+                    _("The net weight of product '%s' must be lower than gross weight.")
```
Comment (rousseldenis) [inline product_net_weight/models/product_product.py:36]: You should use translation function arguments instead.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Little change

File: product_net_weight/models/product_product.py (hunk 1 above)
Comment (alexis-via) [reply product_net_weight/models/product_product.py:36]: It is not required when you have a single variable term

Comment (rvalyi) [reply product_net_weight/models/product_product.py:36]: indeed, there are no warning. Merging this...

**Pull request merged**